browser/src/Services/AutoClosingPairs.ts

```diff
@@ -213,6 +223,40 @@ export const getWhiteSpacePrefix = (str: string): string => {
     }
 }
 
+export const checkOpenCharacter = async (
+    inputFunc: any,
+    pair: IAutoClosingPair,
+    editor: Oni.Editor,
+    openCharacterSameAsClosed: boolean,
+): Promise<void> => {
+    // TODO: PERFORMANCE: Look at how to collapse this instead of needed multiple asynchronous calls.
+
+    const activeBuffer = editor.activeBuffer as IBuffer
+
+    if (openCharacterSameAsClosed) {
+        const lines = await (activeBuffer as any).getLines(
+            activeBuffer.cursor.line,
```

Comment:
For the tests, it looks like `MockBuffer` doesn't have a `cursor` property yet - we may need to add it to get the tests working here / exercise this completely.